aws/aws.go

```diff
@@ -1646,6 +1646,33 @@ func GetAllResources(targetRegions []string, excludeAfter time.Time, resourceTyp
 		}
 		// End CloudWatchAlarm
 
+		//Security Hub
+		securityHub := SecurityHub{}
+		if IsNukeable(securityHub.ResourceName(), resourceTypes) {
+			start := time.Now()
+			hubArns, err := getAllSecurityHubArns(cloudNukeSession, excludeAfter)
+			if err != nil {
+				ge := report.GeneralError{
+					Error:        err,
+					Description:  "Unable to check if Security Hub is enabled",
+					ResourceType: securityHub.ResourceName(),
+				}
+				report.RecordError(ge)
+			}
+			telemetry.TrackEvent(commonTelemetry.EventContext{
+				EventName: "Done Checking if Security Hub is enabled",
```

Comment:
```suggestion
				EventName: "Done Listing Security Hub",
```
The telemetry in mixpanel is searching for events beginning with "Done Listing"